lib/donations/currencies.rb

```diff
@@ -0,0 +1,47 @@
+# == Currency Converter to JSON
+#
+# Takes an array of integer amounts, in US cents, and
+# converts to GBP and EUR.
+#
+# ==== Examples
+#
+#   currencies = Donations::Currencies.for(102, 204, 302, 401)
+#   currencies.to_hash # also to_json
+#   =>
+#   {
+#     USD: [1.02, 2.04, 3.02, 4.01],
```

Comment:
One catch to rounding in JS that is worth thinking about and could be a little tricky is that we'll want to merge values which are identical. For instance, `$1` and `$3` could both evaluate to `£2` if the conversion rates are working right and we round to the closest whole number. Again, just an example, but we wouldn't want our first two values in the donation UX to be `£2 £2`, but rather just a single `£2`. 

Could be a bit tricky, but it's an edge case we definitely need to handle. We might be able to just always go up one, but that's a call for @paulaferris.